_tabs/principles.md

```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
+
+- What to avoid
+   - Overcommitting to too many features at once.
+   - Spending time on polishing non-essential elements or perfecting UI components before validating the core.
+   - Failing to clearly define what’s in or out of scope, leading to scope creep.
 
 ## We ship early
-   - Getting feedback as early as possible even if the project/feature isn’t 100% there
-   - While we control radius with feature flags to reduce risk and deliver value to our customers
+_"Ship Sooner, Learn Faster: Early delivery for quicker feedback."_
+
+- What to do
+   - Releasing minimum viable products (MVPs) to get feedback from users quickly.
+   - Setting aggressive but realistic release targets to avoid endless development cycles.
+   - Prioritizing time-to-market over exhaustive development, even if it means the first version has known limitations.
+
+- What to avoid
+   - Waiting for a product to be "perfect" before releasing it.
+   - Ignoring early customer feedback or postponing releases due to minor issues.
+   - Spending too much time on internal polishing without shipping anything.
 
-## We iterate and improve
-   - Once you got the feedback, go back and improve. As we ship and iterate, we plan for the future as well
-   - We measure what we ship to iterate and improve based on data
+## We learn, iterate, and improve
+_"Learn Fast, Adapt Faster: Every step teaches us something new.""_
+
+- What to do
+   - Analyzing feedback from users or customers after each release and using it to shape future iterations.
+   - Running experiments and collecting data to inform decisions.
+   - Regularly conducting retrospectives to identify what worked well and where improvements are needed.
+
+- What to avoid
+   - Sticking rigidly to the original plan without adjusting based on new information.
+   - Ignoring feedback or failing to collect it after each iteration.
+   - Treating a release as a final solution rather than a stepping stone to the next version.
 
 ## We take ownership
-   - Owning the code we shipped as we are the expert and knows best
-   - Sharing your knowledge by documenting, pairing and training, because sharing is caring
+_"Own It, Lead It: Great outcomes start with taking responsibility."_
+
+- What to do
+   - Owning the entire lifecycle of our code, from development to production and maintenance.
+   - Actively engaging in problem-solving, taking responsibility for identifying and fixing issues rather than passing them off.
+   - Communicating proactively about project status, blockers, or failures, and following through on commitments. 
+
+- What to avoid
+   - Blaming other teams or external factors for project delays or failures.
+   - Failing to address production issues promptly or waiting for someone else to solve the problem.
+   - Over-relying on PMs or others for validation without taking responsibility for writing CI tests.
```

Comment:
Over-relying on PMs or others for validation without taking responsibility for writing CI tests or reviewing own deployed chunks.